index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="index2.css" />
+  </head>
+  <body>
+    <section class="first-section">
+      <div class="first-container">
+        <nav class="first-nav">
+          <h1 class="my-logo">My Logo</h1>
+          <div class="hamburger">
+            <img src="./images/Union.svg" alt="union background" />
+          </div>
+        </nav>
+
+        <h2 class="first-para">
+          I’m Mirela<br />
+          Glad to see <br />
+          you!
+        </h2>
+        <p class="second-para">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <p class="third-para">LET’S CONNECT</p>
```

Comment:
- [ ] [REQUIRED] The subtitle should use a heading tag `hx` (examples: `<h2>`,`<h3>`,`<h4>`, etc...). please, replace the `<p>` with a heading tag (I suggest `<h2>`) ♻️ .